### PR TITLE
Add AGL 13 Marlin to DomD

### PR DESCRIPTION
**Pull request by rshym**
Introduce DomD with AGL v13 Marlin

### CODE REVIEW

**Comment by rshym** [comment]
Tested on h3ulcb-4x2g and h3ulcb-4x2g-kf.
Rebased to the latest prod-devel-rcar

File: meta-agl-tune/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+This layer contains files that were:
+ - copied as-is or adopted from AGL layer meta-agl/meta-agl-bsp/meta-rcar-gen3;
+ - developed by EPAM.
+
+We can't use meta-agl-bsp due to lots of conflicts in handling of prebuilt
+binaries and duplication of functionality with out layers.
```
Comment (lorc) [inline meta-agl-tune/README.md:6]: "our layers", maybe?

File: prod-devel-rcar-agl.yaml [hunk 2]
```diff
@@ -0,0 +1,800 @@
...
+  XT_PVR_NUM_OSID: "2"
+  XT_OP_TEE_FLAVOUR: "generic_dt"
+  XT_GENERIC_DOMU_TAG: ""
+  XT_DOMA_TAG: ""
+  XT_DOMA_DDK_KM_PREBUILT_MODULE: ""
+  XT_DOMA_KERNEL_EXTRA_MODULES: ""
+  XT_DOMA_SOURCE_GROUP: ""
+  XT_DOMZ_TAG: ""
+  # For this product we use one config for all machines for Zephyr
+  XT_DOMZ_CONFIG_NAME: "domz-generic.cfg"
```
Comment (lorc) [inline prod-devel-rcar-agl.yaml:25]: Do we really need Zephyr support in this project?

File: meta-agl-tune/README.md (hunk 1 above)
Comment (rshym) [reply meta-agl-tune/README.md]: Fixed in the text and in the commit message.

File: prod-devel-rcar-agl.yaml (hunk 2 above)
Comment (rshym) [reply prod-devel-rcar-agl.yaml]: Zephyr-related blocks are removed.

**Comment by rshym** [comment]
From my point of view, `meta-agl-tune` is product-specific and even specific for `-agl.yaml` provided by this PR.
It is the result of using the snapshot of AGL, as implemented in this PR.

**Comment by lorc** [comment]
Okay, let it be in this repo. If there will be another AGL-based product, we'll consider moving it into some common location.

`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Pull request merged**